validation/superchain-version_test.go

```diff
@@ -32,6 +33,21 @@ func TestSuperchainWideContractVersions(t *testing.T) {
 		desiredSemver, err := SuperchainSemver[superchain.Superchain].VersionFor("ProtocolVersions")
 		require.NoError(t, err)
 		checkSemverForContract(t, "ProtocolVersions", superchain.Config.ProtocolVersionsAddr, client, desiredSemver)
+
+		isExcludedFromSuperchainConfigCheck := map[string]bool{
+			"Goerli":       true, // no version specified
+			"Goerli Dev 0": true, // no version specified
+			"Mainnet":      true, // no version specified
```

Comment:
Any superchain not in this exclusion list will fail the check if they fail to specify a SuperchainConfig semver.